dsl/pace/dsl/dace/build.py

```diff
@@ -37,198 +30,75 @@ def unblock_waiting_tiles(comm, sdfg_path: str) -> None:
             comm.send(sdfg_path, dest=tile * tilesize + comm.Get_rank())
 
 
-def top_tile_rank_from_decomposition_string(string, partitioner):
-    """
-    Return the rank number on the correct subtile position by matching
-    the decomposition string and the position given by the partitionner
-        e.g.: return rank for "00" for bottom left subtile
-    """
-    tilesize = partitioner.total_ranks / 6
-    if tilesize == 1:
+def get_target_rank(rank: int, partitioner: TilePartitioner):
```

Comment:
I will brief @twicki on the new way I do stuff (got rid of decomposition.yml!), but yes, it can (and actually it does set everything up already, just only when orchestrated -_o_-)